pom.xml

```diff
@@ -31,6 +31,24 @@
       </license>
     </licenses>
 
+    <repositories>
+      <repository>
+        <id>repo.jenkins-ci.org</id>
+        <url>http://repo.jenkins-ci.org/public/</url>
+      </repository>
+      <repository>
+          <id>jitpack.io</id>
+          <url>https://jitpack.io</url>
+      </repository>
+    </repositories>
+
+    <pluginRepositories>
+      <pluginRepository>
+        <id>repo.jenkins-ci.org</id>
+        <url>http://repo.jenkins-ci.org/public/</url>
+      </pluginRepository>
+    </pluginRepositories>
+
```

Comment:
The repositories are already contained in the parent POM, so there is no need to redefine them.